### ISSUE TITLE
Poor image reconstruction quality.

### ISSUE THREAD

**Issue by zhaoyk1986** [vladmandic/dcae#4]
Great work. I simplified your implementation by reducing the number of channels. I used around 1,000 images from the FFHQ dataset for reconstruction training. The loss function is MSE. The training proceeded normally, and the loss decreased as expected, but the resulting images are quite blurry. I would like to know where my code can be improved. If possible, sharing your training experience would be even better. Looking forward to your response.

![无标1题](https://github.com/user-attachments/assets/97c01909-5346-4d9b-b1c4-61f21bc2c385)
![无标题](https://github.com/user-attachments/assets/ad7ebb29-4a06-472d-9f18-0b8b10c875ab)

**Comment by zhaoyk1986** [comment]
[nets.zip](https://github.com/user-attachments/files/18134070/nets.zip)
this is the files

**Comment by SwayStar123** [comment]
@zhaoyk1986 Im not downloading that zip file. Why dont you upload the code to github? 

If you decreased channels while keeping the compression factor the same (eg f=32, c=16), then the compression rate will increase from 96x to 192x. Due to this bottleneck your model may be outputting blurry images.

**Comment by zhaoyk1986** [comment]
@SwayStar123 

Thank you for your reply.
Your answer is very inspiring. My original intention was to train the model using a small amount of data on a reduced channel. The model is likely to experience overfitting.
However, the fact is that the loss is decreasing, but the images are still somewhat blurry.
It is possible that, as you mentioned, the compression ratio of the model is too high, resulting in the inability to reconstruct clear images.
